velox/exec/tests/WindowTest.cpp

```diff
@@ -79,6 +79,74 @@ TEST_F(WindowTest, spill) {
   ASSERT_GT(stats.spilledPartitions, 0);
 }
 
+TEST_F(WindowTest, rankLikeWithEqualValue) {
```

Comment:
Why does the test name have rank in it when you are testing with function sum ?

Reply:
@aditi-pandit Update.